src/main/java/racingcar/service/CarService.java

```diff
@@ -36,10 +39,10 @@ public List<String> splitCarNames(String carNames) {
         return List.of(carNames.split(InputConstant.CAR_NAME_SPLIT_DELIMITER.getMessage()));
     }
 
-    public LinkedHashMap<String, Integer> runRound() {
-        LinkedHashMap<String, Integer> roundResult = new LinkedHashMap<String, Integer>();
+    public Map<String, Integer> runRound() {
+        Map<String, Integer> roundResult = new LinkedHashMap<>();
+        Random random = new Random();
         for (Car car : cars.getCarInformation()) {
-            Random random = new Random();
             int randomNumber = random.nextInt(RANDOM_NUM_MAX_VALUE);
             runForward(car, randomNumber);
```

Comment:
Car 객체에게 메시지를 던져서, Car 자신이 전진을 위한 값을 판단하고, 전진할 수 있으면 전진시키도록 변경해보는건 어떤가요? 😄 

```suggestion
            car.runForward(randomNumber)
```

Reply:
리팩토링 과정에서 CarService.java가 Car.java가 가져야하는 책임을 다소 가지고 있었다는 것을 알게되었습니다. 
**RacingcarController**에서 **Cars**에게 메시지를 던지고, **Cars**에서 **Car**로 **랜덤값과 함께 메세지를 던져** Car 에서 스스로 값을 판단하고 전진 할 수 있도록 리팩토링해보았습니다 🥺